external/fv3fit/fv3fit/data/netcdf/io.py

```diff
@@ -12,7 +12,8 @@
 logger = logging.getLogger(__name__)
 
 
-def download_cached(path):
+def download_cached(path: str, cache: str = CACHE_DIR) -> str:
```

Comment:
This function needs to be updated to use this argument.

Reply:
good catch!